### PR TITLE
PyPI Related Changes

### LINKED ISSUE #35: Documentation in Jupyter Book
Create comprehensive documentation using Jupyter Book.

### PR DESCRIPTION
**Pull request by Sohambutala**
PyPI related changes and Pydantic upgrade.

Fix #35 #36 #38

### CODE REVIEW

**Comment by leewujung** [comment]
@Sohambutala : Could you change the PR title to something that describes the changes? Thanks!

**Comment by Sohambutala** [comment]
@leewujung : Sorry, did not realize that, I was fiddling with dev branch deployment and releasing on PyPI, but unfortunately did not work.

**Comment by leewujung** [comment]
Oh no worries, I happened to see this in my inbox so decided to comment on it!

File: jupyterbook/aws/pipelineconfiguration.md
Comment (valentina-s) [inline jupyterbook/aws/pipelineconfiguration.md:1]: Make title to indicate this is an AWS example

**Comment by valentina-s** [comment]
Looks good @Sohambutala. Could you also list the issues it addresses. Eventually, they will close themselves when the PR gets merged. And assign a reviewer when the PR is ready, then we will receive an email notification.

**Review by valentina-s — APPROVED** [review]
Looks good @Sohambutala. Could you also list the issues it addresses. Eventually, they will close themselves when the PR gets merged. And assign a reviewer when the PR is ready, then we will receive an email notification.

**Pull request merged**